src/mono/mono/metadata/image-internals.h

```diff
@@ -9,6 +9,17 @@
 #include <mono/metadata/image.h>
 #include <mono/metadata/loader-internals.h>
 
+typedef struct {
+	int care_about_cli : 1;
+	int care_about_pecoff : 1;
```

Comment:
let's negate these: seems like most of the time we care about cli or pecoff.
```suggestion
	int dont_care_about_cli : 1;
	int dont_care_about_pecoff : 1;
```

This way most of the time just:
```c
   MonoImageOpenOptions options = {0,};
```
will initialize to a nice default.

Reply:
Done!